guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -321,6 +346,83 @@ A new Content View called `Import-Library` is created in the target organization
 This Content View is used to facilitate the library content import.
 
 
+
+=== Example Scenarios
+
+The import and export functionality can be used mainly in the following scenarios:
+
+* If you want to copy some but not all content from your {ProjectServer} to other {ProjectServer}s.
+For example, you have Content Views that your IT department validates on {ProjectServer}, and you want to copy content from those Content Views to other {ProjectServer}s.
+
+* If you want to copy all library content from your {ProjectServer} to other {ProjectServer}s.
+For example, you have Products and repositories that your IT department validates on {ProjectServer} in the Library, and you want to copy all Products and repositories in that organization to other {ProjectServer}s.
+
+* If you have both connected and disconnected {ProjectServer}s, and want to copy content from the connected servers to the disconnected servers.
+For example, you require complete isolation of your management infrastructure for security or other purposes.
+
+These scenarios require at least two {ProjectServer}s: one {ProjectServer} that is connected to the outside world and contains content to export, and another {ProjectServer} that is disconnected that you want to import content to.
+
+[[img-disconnected]]
+ifdef::satellite[]
+image::Disconnected.png[title="The {ProjectNameX} ISS use cases", alt="The {ProjectNameX} ISS use cases"]
+endif::[]
+==== Air gapped
+
+In this scenario, you have one connected {ProjectServer} that is receiving content from an external source like the CDN.
+The rest of your infrastructure is completely isolated, including another disconnected {ProjectServer}.
+The connected {ProjectServer} is mainly used as a content store for updates.
+The disconnected {ProjectServer} serves as the main {ProjectServer} for managing content for all infrastructure behind the isolated network.
+
+.On the connected {ProjectServer}
+
+. Set the`Default Red Hat Repository download policy` to *Immediate* (see xref:Importing_Content-Configuring_Download_Policies[].)
+. Enable the content that you require.
```

Comment:
it might make more sense to just say:

* ensure that repositories are using the immediate download policy, either by:
  *   For existing repos using "On Demand" change their download policy on the repository details page to 'immediate' and resync
  * For new repositories, ensure that the 'Default Red Hat Repository download policy' setting is set to 'immediate' before enabling RH repos, and 'Default download policy' is set to immediate for custom repositories

Reply:
+1